<ui version="4.0" >
 <class>FluidCharacteristicsForm</class>
 <widget class="QWidget" name="FluidCharacteristicsForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5" >
   <item>
    <widget class="QGroupBox" name="groupBoxRho" >
     <property name="title" >
      <string>Density</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4" >
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>53</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" >
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" >
            <item>
             <widget class="QComboBox" name="comboBoxRho" >
              <property name="toolTip" >
               <string>Code_Saturne key word: IROVAR</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>17</width>
                <height>17</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonRho" >
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="resources_pages.qrc" >
                <normaloff>:/new/prefix1/icons/22x22/edit-find-replace.png</normaloff>:/new/prefix1/icons/22x22/edit-find-replace.png</iconset>
              </property>
              <property name="iconSize" >
               <size>
                <width>22</width>
                <height>22</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>13</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" >
          <item>
           <widget class="QLabel" name="labelRho" >
            <property name="text" >
             <string>reference</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>16</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="labelVarRho" >
            <property name="font" >
             <font>
              <family>Symbol</family>
             </font>
            </property>
            <property name="text" >
             <string>r</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditRho" >
            <property name="toolTip" >
             <string>Code_Saturne key word: RO0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelUnitRho" >
            <property name="text" >
             <string>kg/m&lt;sup>3&lt;/sup></string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>53</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxMu" >
     <property name="title" >
      <string>Viscosity</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8" >
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>21</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" >
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6" >
            <item>
             <widget class="QComboBox" name="comboBoxMu" >
              <property name="toolTip" >
               <string>Code_Saturne key word: IVIVAR</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>21</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonMu" >
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="resources_pages.qrc" >
                <normaloff>:/new/prefix1/icons/22x22/edit-find-replace.png</normaloff>:/new/prefix1/icons/22x22/edit-find-replace.png</iconset>
              </property>
              <property name="iconSize" >
               <size>
                <width>22</width>
                <height>22</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_2" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>13</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5" >
          <item>
           <widget class="QLabel" name="labelMu" >
            <property name="text" >
             <string>reference</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>16</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="labelVarMu" >
            <property name="font" >
             <font>
              <family>Symbol</family>
             </font>
            </property>
            <property name="text" >
             <string>m</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditMu" >
            <property name="toolTip" >
             <string>Code_Saturne key word: VISCL0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelUnitMu" >
            <property name="text" >
             <string>Pa.s</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>21</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxCp" >
     <property name="title" >
      <string>Specific heat</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_12" >
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>50</width>
          <height>22</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" >
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10" >
            <item>
             <widget class="QComboBox" name="comboBoxCp" >
              <property name="toolTip" >
               <string>Code_Saturne key word: ICP</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>13</width>
                <height>22</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonCp" >
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="resources_pages.qrc" >
                <normaloff>:/new/prefix1/icons/22x22/edit-find-replace.png</normaloff>:/new/prefix1/icons/22x22/edit-find-replace.png</iconset>
              </property>
              <property name="iconSize" >
               <size>
                <width>22</width>
                <height>22</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_3" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>17</width>
              <height>17</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9" >
          <item>
           <widget class="QLabel" name="labelCp" >
            <property name="text" >
             <string>reference</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>13</width>
              <height>22</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="labelVarCp" >
            <property name="text" >
             <string>Cp</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditCp" >
            <property name="toolTip" >
             <string>Code_Saturne key word: CP0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelUnitCp" >
            <property name="text" >
             <string>J/kg/K</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>50</width>
          <height>22</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxAl" >
     <property name="title" >
      <string>Thermal conductivity</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_16" >
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>53</width>
          <height>22</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4" >
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14" >
            <item>
             <widget class="QComboBox" name="comboBoxAl" >
              <property name="toolTip" >
               <string>Code_Saturne key word: IVISLS</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>13</width>
                <height>22</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonAl" >
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="resources_pages.qrc" >
                <normaloff>:/new/prefix1/icons/22x22/edit-find-replace.png</normaloff>:/new/prefix1/icons/22x22/edit-find-replace.png</iconset>
              </property>
              <property name="iconSize" >
               <size>
                <width>22</width>
                <height>22</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_4" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>13</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13" >
          <item>
           <widget class="QLabel" name="labelAl" >
            <property name="text" >
             <string>reference</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>13</width>
              <height>22</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="labelVarAl" >
            <property name="font" >
             <font>
              <family>Symbol</family>
             </font>
            </property>
            <property name="text" >
             <string>l</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditAl" >
            <property name="toolTip" >
             <string>Code_Saturne key word: VISLS0*CP0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelUnitAl" >
            <property name="text" >
             <string>W/m/K</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>52</width>
          <height>22</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxRho</tabstop>
  <tabstop>lineEditRho</tabstop>
  <tabstop>comboBoxMu</tabstop>
  <tabstop>lineEditMu</tabstop>
  <tabstop>comboBoxCp</tabstop>
  <tabstop>lineEditCp</tabstop>
  <tabstop>comboBoxAl</tabstop>
  <tabstop>lineEditAl</tabstop>
 </tabstops>
 <resources>
  <include location="resources_pages.qrc" />
 </resources>
 <connections/>
</ui>
